tests/specs/complete.yaml

```diff
@@ -55,10 +55,11 @@ paths:
       requestBody:
         required: true
         description: The event to publish.
-        application/octet-stream:
-          schema:
-            type: string
-            format: binary
+        content:
```

Comment:
The previous file we used for integration tests had invalid OpenAPI syntax.
![image](https://github.com/asecurityteam/transportd/assets/8324854/3bed7667-80c7-481b-b69d-d30129b15ee6)